### PR TITLE
Group38  Dplyr Relational Databases

### PR DESCRIPTION
**Pull request by derekcoding1**
Contribution of Group 38

### CODE REVIEW

File: group38.Rmd [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+                      and Debt_service_filter.date = Property_prices.date
+                      ")
+glimpse(innerjoin_sqldf)
+```
+
+
+### 5.4. Full_join
+
+
+![](full_join.png){width=25%}
```
Comment (bojum) [inline group38.Rmd]: Should this be resources/group38/full_join.png

**Review by bojum — DISMISSED** [review]
Your rmd file cant access the pictures because it has the wrong paths.

**Comment by skyetim** [comment]
name of your `Rmd` file is also not explanatory. please refer to the instructions.

**Comment by bojum** [comment]
🎉 Your work has been featured on the class website [here](https://jtr13.github.io/cc19/dplyr-relational-databases.html). Thanks!

**Pull request merged**